_rules/aria-state-or-property-permitted-5c01ea.md

```diff
@@ -141,10 +167,20 @@ No [WAI-ARIA state or property](https://www.w3.org/TR/wai-aria-1.1/#state_prop_d
 
 #### Inapplicable Example 2
 
-`aria-sort` property is neither inherited, supported, nor required for role `button`, but the element is not [included in the accessibility tree][].
+`aria-sort` property is neither inherited, supported, nor required for role `button`, but the element is not [included in the accessibility tree][] because it is not rendered.
```

Comment:
```suggestion
`aria-sort` property is neither inherited, supported, nor required for role `button`. In addition, the element is not [included in the accessibility tree][] because it is hidden to everybody using CSS.
```
I think the element is rendered, though it is not displayed because of the `display:none;` style. Anyway, I think it helps clarify the description.